script.lua

```diff
@@ -3226,57 +3225,96 @@ function setupPlayerArea(params)
 
     local energy = 0
 
-    local function elemStrToArr(elemStr)
-        local outArr = {}
-        for i = 1, string.len(elemStr) do
-            table.insert(outArr,(math.floor(string.sub(elemStr, i, i))))
-        end
-        return outArr
-    end
-
-    local function elemCombine(inTableOfElemStrCards)
+    local Elements = {}
+    Elements.__index = Elements
+    function Elements:new(init)
```

Comment:
from what i can tell init is never passed in here, can that part just be removed?

Reply:
It is used in #75.